Mathlib/AlgebraicGeometry/ProjectiveSpectrum/StructureSheaf.lean

```diff
@@ -124,16 +124,16 @@ theorem addMem' (U : (Opens (ProjectiveSpectrum.top 𝒜))ᵒᵖ) (a b : ∀ x :
       fun y => _⟩
   simp only at wa wb
   simp only [Pi.add_apply, wa ⟨y.1, y.2.1⟩, wb ⟨y.1, y.2.2⟩, ext_iff_val,
-    add_val, val_mk'', add_mk, add_comm (sa * rb)]
+    val_add, val_mk, add_mk, add_comm (sa * rb)]
   rfl
 #align algebraic_geometry.projective_spectrum.structure_sheaf.section_subring.add_mem' AlgebraicGeometry.ProjectiveSpectrum.StructureSheaf.SectionSubring.addMem'
 
 theorem negMem' (U : (Opens (ProjectiveSpectrum.top 𝒜))ᵒᵖ) (a : ∀ x : U.unop, at x.1)
```

Comment:
Since we are doing refactoring any way, should this be named `neg_mem’` instead, similarly for `mulMem’`